src/main/java/com/hackclub/hccore/commands/LocCommand.java

```diff
@@ -113,11 +115,46 @@ public boolean onCommand(CommandSender sender, Command cmd, String alias, String
                             + "Please specify the location name and the player you want to share it with");
                     break;
                 }
-                if (!data.getSavedLocations().containsKey(args[1])) {
+                locationName = args[1];
+                String recipientName = args[2];
+                
+                if (!data.getSavedLocations().containsKey(locationName)) {
                     sender.sendMessage(ChatColor.RED + "No location with that name was found");
                     break;
                 }
+                Location sendLocation = data.getSavedLocations().get(locationName);
                 // TODO: Add share functionality
+                // Get the player we're sending to
+                Player recipient = org.bukkit.Bukkit.getPlayer(recipientName);
+                if (recipient == null) {
+                    sender.sendMessage(ChatColor.RED
+                            + "Player '" + recipientName + "' does not exist.");
```

Comment:
Change it to "No online player with that name was found" to make it consistent with the other commands.